docs/input_formats/if_landxml.rst

```diff
@@ -0,0 +1,195 @@
+| Property         |                    | Feature          |                          |
++------------------+--------------------+------------------+--------------------------+
+|                  | values             |                  | attrib                   |
++------------------+--------------------+------------------+--------------------------+
+| InstrumentSetup  |                    | Survey           | "ST" in feature          |
++------------------+--------------------+------------------+--------------------------+
+|                  | id                 |                  | station_id               |
++------------------+--------------------+------------------+--------------------------+
+|                  | stationName        |                  | point_name               |
++------------------+--------------------+------------------+--------------------------+
+|                  | instrumentHeight   |                  | ih                       |
++------------------+--------------------+------------------+--------------------------+
+|                  | orientationAzimuth |                  | hz0                      |
++------------------+--------------------+------------------+--------------------------+
+|                  | circleAzimuth      |                  | hz0                      |
++------------------+--------------------+------------------+--------------------------+
+| Feature          |                    | InstrumentSetup  |                          |
++------------------+--------------------+------------------+--------------------------+
+| Property         |                    | Feature          |                          |
++------------------+--------------------+------------------+--------------------------+
+|                  | values             |                  | attrib                   |
++------------------+--------------------+------------------+--------------------------+
+| ObservationGroup |                    | Survey           |                          |
++------------------+--------------------+------------------+--------------------------+
+|                  | setupID            |                  | station_id               |
++------------------+--------------------+------------------+--------------------------+
+| Backsight*       |                    | ObservationGroup |                          |
++------------------+--------------------+------------------+--------------------------+
+|                  | circle             |                  | circle                   |
++------------------+--------------------+------------------+--------------------------+
+|                  | setupID            |                  | "setup" + id             |
++------------------+--------------------+------------------+--------------------------+
+| BacksightPoint*  |                    | ObservationGroup |                          |
```

Comment:
I thought you implemented Backsight in the other PR ?

Reply:
Hahahah yes, I write documentation before the change ^^
Backsight was already here but not on the right place.